.github/actions/build/containers-push/action.yml

```diff
@@ -0,0 +1,138 @@
+name: "Push Containers"
+description: "Pushes container images and creates manifests"
+
+inputs:
+  architectures:
+    description: "Comma-separated list of architectures (e.g., 'amd64,arm64,s390x,ppc64le')"
+    required: true
+  runnerArch:
+    description: "Runner architecture (amd64, arm64)"
+    required: false
+    default: "amd64"
+  quayUser:
+    description: "Quay.io username"
+    required: true
+  quayPass:
+    description: "Quay.io password"
+    required: true
+  cosignPassword:
+    description: "Cosign password for signing"
+    required: true
+  cosignPrivateKey:
+    description: "Cosign private key for signing"
+    required: true
```

Comment:
I think cosign can work in a better mode in GitHub actions. We can leave that for later, but we should at least have some TODO.

Reply:
I found this https://github.com/sigstore/cosign-installer and this https://github.com/marketplace/actions/gh-action-sigstore-python . The second option might be usable, but it for sure need more investigation. I can use the installer instead of our script and leave TODO for some followup PR to see if we can use the other action for signing.